pkg/skaffold/schema/validation/validation.go

```diff
@@ -457,3 +458,16 @@ func validateLogPrefix(lc latest.LogsConfig) []error {
 
 	return nil
 }
+
+func validateSingleKubeContext(configs []*latest.SkaffoldConfig) []error {
```

Comment:
hmm. is it possible to support per-config kube context? i'm not saying we should do it now (that code is a mess), but definitely something to think about. i could imagine scenarios where different components are deployed to different clusters, though maybe this is out of scope for the skaffold workflow.

Reply:
yes, I have left a TODO for that if we ever get to it. We can remove this validation then.